### ISSUE TITLE
When I make comment sometimes I get "Item Not Stored", but retry works so not blocked.

### ISSUE THREAD

**Issue by 4nanook** [friendica/friendica#10266]
- [ x] I have searched open and closed issues for duplicates

### Bug Description

When I comment on someone else's post or on someone else's comment, I sometimes get a little brown pop-up in the upper right corner that says, "Item Wasn't Stored", but retrying the exact same comment usually works so I am not blocked or the second attempt would fail in the same manner.  This seems to be a problem more with posts originating on diaspora servers than any others.
### Steps to Reproduce

1.  step one
2.  step two
3.  step three
Try to post a comment.
Actual Result:

A little brown pop-up with the words "Item Wasn't Stored" and no further explanation.
Expected Result:

The comment should appear in line the posts and/or other comments.
### Screenshots

Problem description is simple enough that a screen shot did not seem warranted.  If you really need one let me know.
### Platform Info

Friendica Version:

This is Friendica, version 2021.06-dev that is running at the web location https://friendica.eskimo.com. The database version is 1414/1414, the post update version is 1400/1400.
Friendica Source: 
Github
PHP version:
PHP 7.4.18 (cli) (built: May  3 2021 11:27:06) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.18, Copyright (c), by Zend Technologies

SQL version:
mysql  Ver 15.1 Distrib 10.5.10-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**Comment by AlfredSK** [comment]
Confirmed. Same here. Happens every now and then. 
2021.06-dev

**Comment by MrPetovan** [comment]
No reliable way of reproducing, likely depending on underlying infrastructure, removing from milestone.

**Comment by 4nanook** [comment]
Ok, I've got Hubzilla, about fifty wordpress sites, squirrelmail, roundcube, raintree, and nextcloud, and ALL of them work reliably, ONLY friendica behaves in this intermittent fashion, both with storing comments and with occasionally not using CSS properly, NO OTHER APPLICATION does this.  So I debate your infrastructure argument.

**Comment by MrPetovan** [comment]
I'm pretty sure it has to do with the max load setting that none of the other services have. This max load setting should be relative to the server's capacity, however we can only provide one default value for everybody (or turn the setting off by default which could also be sensible). Please check the Maximum system load setting values in the Admin settings.

**Comment by 4nanook** [comment]
![maximums](https://user-images.githubusercontent.com/3466797/132273464-5123c1ab-3309-4cda-85b2-d5ac6549800f.png)

There are my max load settings.  Here is some info on the machine:
```
free -h
              total        used        free      shared  buff/cache   available
Mem:           62Gi        41Gi       3.9Gi       125Mi        17Gi        20Gi
Swap:         127Gi       2.4Gi       125Gi
```

As you can see there are 20G available so requiring 1G free should not be unreasonable.  Please feel free to request any
other info you may need.  The database is on an iomemory flash drive as recommended for maximum performance by the MariaDB people.  It has successfully handled 10,000+ transactions per second though it only averages 200-300 under normal circumstances (those high TPS values tend to happen if I've had the machine down for several hours and then bring it back up, everything trying to sync with various networks).

**Comment by MrPetovan** [comment]
What's the output of `uptime` on your server? What I'm talking about isn't the memory but the load average, a measure of how busy a server is. It's the average number of processes waiting for CPU time at any given moment. When all computers had only one CPU, this number meant the same thing on all of them, but since the creation of multi-core CPUs, this value means very different things on very different systems.

A value of 40 on a single-core CPU means that the computer is severely lagging with too many concurrent processes fighting over very limited computing resources, while the same number on a 8 or 16 cores CPU as it is now common on high-end server architecture would mean normal operation.

**Comment by 4nanook** [comment]
uptime
 19:52:54 up 1 day, 14:43,  1 user,  load average: 1.36, 2.27, 3.17

This is a six core twelve thread i7-6850 server clocked at 4.2 Ghz.

**Comment by MrPetovan** [comment]
Thanks, this is the load average for the worker, you should raise it to 80 but it won't change the frontend max load average which is in "Advanced". This value should also be raised to 80 for your system if you don't experience slow loading times in Friendica.

**Comment by 4nanook** [comment]
Ok, I've set both to 80, we'll see how things work.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by 4nanook** [comment]
Only time load ever gets above 40 is after a reboot when everything is starting up so not really convinced this is the issue but I've set to 80 anyway.

**Comment by MrPetovan** [comment]
Even a passing load of 40 would trigger the Friendica limit and on a 6 core system the load varies more than with fewer cores so I'm confident it could help.

**Comment by 4nanook** [comment]
Well hate to say it but I just hit home and it loaded the HTML but NOT the CSS again.

**Comment by MrPetovan** [comment]
I'm sorry about that, to troubleshoot further I'd need at least the HTTP error code returned by the CSS stylesheet request.

**Comment by 4nanook** [comment]
Well that's the odd bit, there are NO errors in the logs associated with friendica loading CSS.  And CSS loading NEVER fails in Wordpress.

**Comment by MrPetovan** [comment]
CSS are dynamic in Friendica which means it's a PHP script outputting CSS rules. So it definitely can fail more than loading a static file like in Wordpress.

**Comment by 4nanook** [comment]
Ok, is there anything logged when load limits come into play so I can see if this is an issue?  This has been a problem since day one back in January even when the machine was not heavily trafficked.  Now it's got more than 500 users so traffic is picking up but the frequency of this really has not changed.  I will be putting together a new machine sometime later this year, waiting to see what 7nm Intel chips look like when they come out this October verses AMD before I decide which way to go, going to get two flash cards and RAID them for the database, don't really feel good having it on unraided flash but this motherboard had no flash slots so had to install a card.

**Comment by MrPetovan** [comment]
Yes, it should show `system load for process too high.` in `Warning` level in the Friendica log if you have it enabled in the admin section.

**Comment by 4nanook** [comment]
I've got 230,000+ lines in my log but that message does not show up.  Also tried grep -i in case case was different.

**Comment by MrPetovan** [comment]
Good call on the `grep -i` even if I provided you the exact case we employ. What log level are you currently using for your Friendica log?

**Comment by 4nanook** [comment]
I just checked that only logging errors, just switched that to warning and zero'd the log so only new stuff will show up.  Now will wait for it to screw up again and see if that shows in the logs.

**Comment by MrPetovan** [comment]
Thanks, you did the right thing.

**Comment by 4nanook** [comment]
Of course since doing that it hasn't screwed up once :)  A watched pot never boils.

**Comment by 4nanook** [comment]
Well it just did the fail to load CSS thing, I checked the logs and no "system load for process too high" messages.

**Comment by 4nanook** [comment]
Ah I DID get "worker memory limit too high", I wonder if I forgot to increase the memory limit for the php command line.

**Comment by 4nanook** [comment]
No, I have memory limit set to 4GB, that really should be adequate for any script that isn't broken.

**Comment by 4nanook** [comment]
Hmm, it's always this line:
2021-09-07T09:31:16Z worker [WARNING]: Minimal memory reached. {"free":951,"memtotal":64317,"limit":"1024"} - {"file":"Process.php","line":193,"function":"isMinMemoryReached","uid":"0e90fe","process_id":2862017}
So I have minimal memory free set to 1024 server wide but the server is showing close to 22GB free.  Don't understand
why this is tripping.

**Comment by MrPetovan** [comment]
I don't think you need this setting either if you aren't experiencing slowness, please set it to 0.
